### PR TITLE
Catchy release rosinstall update

### PR DESCRIPTION
**Pull request by stonier**

### CODE REVIEW

File: rosinstalls/indigo/gopher_bootstrap-stable.rosinstall [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
+- git: { local-name: 'elevator_interactions', version: 'release/0.3-indigo',        uri: 'https://bitbucket.org/yujinrobot/elevator_interactions.git'}
 
-# until fixed version of audio has been released
-- git: { local-name: 'audio_common',          version: 'aab0edb3',           uri: 'https://github.com/yujinrobot/audio_common.git'}
-# until I know why the 1.8.10 deb version doesn't properly unload while the 1.8.10 source version does (debug vs release?)
-- git: { local-name: 'diagnostics',           version: '1.8.10',             uri: 'https://github.com/ros/diagnostics.git'}
+# Until https://github.com/ros-infrastructure/rosdoc_lite/pull/70 is in
+- git: { local-name: 'rosdoc_lite',           version: 'master',       uri: 'https://github.com/yujinrobot/rosdoc_lite.git'}
+# Until we've moved off of indigo (our upstream changes are in, but not in indigo release)
+- git: { local-name: 'audio_common',          version: 'master',       uri: 'https://github.com/yujinrobot/audio_common.git'}
```
Comment (hughie) [inline rosinstalls/indigo/gopher_bootstrap-stable.rosinstall]: I'm not sure about this version. It's same as https://github.com/ros-drivers/audio_common. 
There is no related PR and [deb](http://repositories.ros.org/status_page/ros_indigo_default.html?q=audio_common) is released.

File: rosinstalls/indigo/gopher_desktop-stable.rosinstall [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+
+# Enabling library that keeps rqt gui's open even if the robot goes up and down
+- git: { local-name: 'rqt_wrapper',    version: 'release/0.2-indigo', uri: 'https://github.com/stonier/rqt_wrapper.git'}
+
+# Visualisation for behaviour trees
+- git: { local-name: 'rqt_py_trees',   version: 'release/0.2-indigo', uri: 'https://github.com/stonier/rqt_py_trees.git'}
+
+# A really nice plotting tool!
+# Awaiting https://github.com/ethz-asl/rqt_multiplot_plugin/pull/10
+#- git: { local-name: 'rqt_multiplot_plugin',   version: 'release/0.1-indigo', uri: 'https://github.com/yujinrobot/rqt_multiplot_plugin.git'}
```
Comment (hughie) [inline rosinstalls/indigo/gopher_desktop-stable.rosinstall]: This [PR](http://repositories.ros.org/status_page/ros_indigo_default.html?q=rqt_multiplot) is merged and [deb](http://repositories.ros.org/status_page/ros_indigo_default.html?q=rqt_multiplot)is released. so, I comment this. Is it correct?

**Comment by hughie** [comment]
There are 4 rosinstall files for concert. Should I synch all? Do we use both(ssh and normal)?
```gopher_concert-devel.rosinstall``` ,```gopher_concert-devel.ssh.rosinstall```
```gopher_concert-stable.rosinstall```,```gopher_concert-stable.ssh.rosinstall```

File: rosinstalls/indigo/gopher_bootstrap-stable.rosinstall (hunk 1 above)
Comment (naveedhd) [inline rosinstalls/indigo/gopher_bootstrap-stable.rosinstall]: Yes it's same as the upstream but keeping a fork of this, so if any changes go in upstream, we can test first and then pull it in.

**Comment by asmodehn** [comment]
The ssh ones are for when someone doesn't want to use http or have to enter his password.
One developer just has to setup his key and then authentication is automatic to all repositories. 

However I think no one use it anymore... Setting up SSH keys once can be a hassle for some, so instead most people prefer entering their password everytime.

I would say we should remove the *.ssh.rosinstall to avoid needing to maintain too many files. We can easily recreate them if we eventually need them (for full automation for example).

File: rosinstalls/indigo/gopher_concert-stable.rosinstall [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
 #
-# Rocon stacks forked for gopher development, focus is on rocon for the robot. Focus is on concert for the concert.
+#   Permissions: open to all gopher developers - no rocket science here yet, but usually just the concert developers.
 #
 
-- git: { local-name: 'rocon_concert',      version: 'release/0.8-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_concert.git'}
-- git: { local-name: 'rocon_tutorials',    version: 'release/0.8-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_tutorials.git'}
+# Supporting tools from old Rocon
+- git: { local-name: 'rocon_concert',          version: 'release/0.9-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_concert.git'}
+- git: { local-name: 'rocon_tutorials',        version: 'release/0.8-indigo-gopher', uri: 'https://github.com/robotics-in-concert/rocon_tutorials.git'}
```
Comment (hughie) [inline rosinstalls/indigo/gopher_concert-stable.rosinstall]: Do we need rocon_tutorials for concert?

Comment (stonier) [inline rosinstalls/indigo/gopher_concert-stable.rosinstall]: Yes

**Comment by hughie** [comment]
I have question about these forks for version bumping and tagging. These are same as upstream or specific branch, so it doesn't need to follow our own release version. 

I will just make branch as ```release/X.X-indigo-gopher``` without version bumping and tagging. Is it correct?

```
# Until https://github.com/ros-infrastructure/rosdoc_lite/pull/70 is in
- git: { local-name: 'rosdoc_lite',           version: 'master',       uri: 'https://github.com/yujinrobot/rosdoc_lite.git'}
# Until we've moved off of indigo (our upstream changes are in, but not in indigo release)
- git: { local-name: 'audio_common',          version: 'master',       uri: 'https://github.com/yujinrobot/audio_common.git'}
# until we have fixes for the service unloading problem - https://github.com/ros/diagnostics/issues/50
- git: { local-name: 'diagnostics',           version: 'handling_unloading_properly', uri: 'https://github.com/yujinrobot/diagnostics.git'}
```

**Pull request merged**